docs/app/Examples/views/Statistic/Types/StatisticGroupExample.js

```diff
@@ -0,0 +1,24 @@
+import React, {Component} from 'react';
+import {Statistic} from 'stardust';
+const {Statistics, Label, Value} = Statistic;
```

Comment:
Would this change the syntax for all components that have sub-components? Or just those that need to have "groups" for styling? 

I could see using the `Group` sub-component name with `Statistics` and `Segments` but how would this look for other components such as `Menu` and `List`?

I have seen benefits in developing with aligning JS class names and HTML class names while developing with Stardust components, in that I could easily look through the semantic ui docs and make accurate guesses on how the component is named, but I do agree `<Statistic.Statistics>` does not feel as intuitive as `<Statistic.Group>`